source/voice_control/custom_sentences.markdown

```
@@ -15,22 +15,17 @@ related:
 
 ### Prerequisites
 
-If you have not set up voice control yet, set up the hardware first. For instructions, refer to one of the following tutorials:
+You need a working Assist configuration. If you haven't done so yet, check [Assist's starting page](/voice_control/) to get you ready with your setup.
 
-- [World's most private voice assistant](/voice_control/worlds-most-private-voice-assistant/): Using a classic landline phone
-- [$13 voice assistant for Home Assistant](/voice_control/thirteen-usd-voice-remote/): Using a button with speaker and mic
-- [S3-BOX-3 voice assistant](/voice_control/s3_box_voice_assistant/): Using a small device with speaker, mic, and display
-- [Assist for Apple](/voice_control/apple/): Using your iPhone, Mac, or Apple watch
-- [Assist for Android](/voice_control/android/): Using your Android phone, tablet, or a Wear OS watch
 
```

Markdownlint (0.35.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)